tasks/measurements/ookla_speedtest_cli.py

```diff
@@ -0,0 +1,71 @@
+import time
+import subprocess
+from subprocess import Popen
+import os
+import logging
+import subprocess
+import time
+from netunicorn.base.architecture import Architecture # type: ignore
+from netunicorn.base.nodes import Node # type: ignore
+from netunicorn.base.task import Failure, Task, TaskDispatcher # type: ignore
+from netunicorn.client.remote import RemoteClient, RemoteClientException # type: ignore
+from netunicorn.base import Experiment, ExperimentStatus, Pipeline # type: ignore
+from netunicorn.library.tasks.basic import ShellCommand # type: ignore
+from netunicorn.base.architecture import Architecture # type: ignore
+from netunicorn.base.nodes import Node # type: ignore
+from netunicorn.base.task import Failure, Task, TaskDispatcher # type: ignore
+from netunicorn.base import Result, Failure, Success, Task, TaskDispatcher # type: ignore
+from netunicorn.base.architecture import Architecture # type: ignore
+from netunicorn.base.nodes import Node # type: ignore
+from returns.pipeline import is_successful # type: ignore
+from returns.result import Failure # type: ignore
```

Comment:
Please, normalize and deduplicate the imports. Also, verify that you do not import packages you're not using (e.g. logging, Popen, etc)